pages/common/git-restore.md

```diff
@@ -0,0 +1,17 @@
+# git restore
+
+> Restore working tree files. Requires git version 2.23+.
+> See also `git checkout`.
+> More information: <https://git-scm.com/docs/git-restore/>.
+
+- Restore a file that was accidentally deleted:
```

Comment:
I don't think the "accidentally" is needed here :)

Also, maybe specify that this fetches the file contents from the current commit / HEAD?